### ISSUE TITLE
Is OpenKeyChain not actively maintained anymore? 

### ISSUE THREAD

**Issue by life777eternal** [open-keychain/open-keychain#2736]
Somebody in the K-9 Mail repository believes it isn't. And if so, why not? This was originally a replacement for another GnuPG/PGP/OpenPGP app for Android. 

https://github.com/k9mail/k-9/issues/5662

Thank you.

**Comment by Valodim** [comment]
OpenKeychain is no longer actively developed, as indicated by the large warning text on the top of this repository's readme.

**Comment by life777eternal** [comment]
Ok, I did see that aye. Why is it not being actively developed anymore? Did everyone just give up on it? Or is it just that no developers have any time for it anymore?

Thank you.

**Comment by Valodim** [comment]
Here is the rough explanation I put in the commit message:

```
Both @dschuermann and myself no longer spend a lot of time on this project.

Here's a brief summary of a blog post we never wrote:

* All of OpenKeychain's UI is an anti-feature.
* If the user is doing anything in OpenKeychain's UI, OpenPGP is still doing it wrong.
* OpenKeychain shouldn't be an app, it should be a library.
* Changing OpenKeychain into a library is more work than we are motivated to do in our free time.
```

So it's a mixture of both - we no longer have time, but we also aren't convinced anymore that an external openpgp application is the right way to go for openpgp support with useful usability.

**Comment by life777eternal** [comment]
Alright, thanks for the explanation. That does make sense, and I agree that integrating it on the email app/client would be better, like Mozilla Thunderbird now has had for a while. 

I did find exporting secret keys to be a bit of a pain. It forces them to be encrypted with PGP when I don't need them encrypted. That would only be for sending them though an Insecure email/message. It showed a long number that I had to enter with spaces to decrypt it on my computer, when I'm only saving it on the phone storage and transferring it through a USB connection to my computer.
